src/Compilers/CSharp/Test/Semantic/Diagnostics/OperationAnalyzerTests.cs

```diff
@@ -1016,7 +1016,7 @@ public void M1()
                 Diagnostic(VariableDeclarationTestAnalyzer.TooManyLocalVarDeclarationsDescriptor.Id, "int g1, g2, g3, g4 =;").WithLocation(13, 9));
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/8699")]
```

Comment:
Will unskip.

---
In reply to: [70328262](https://github.com/dotnet/roslyn/pull/12416#discussion_r70328262) [](ancestors = 70328262)